collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -467,7 +467,12 @@ private void setupViewModels() {
                 .get(FormEntryViewModel.class);
 
         formEntryViewModel.getCurrentIndex().observe(this, index -> {
-            formIndexAnimationHandler.handle(index);
+            if (index != null && Collect.getInstance().getFormController() == null) {
```

Comment:
The dialog would not show for [this instance of the crash](https://console.firebase.google.com/u/2/project/api-project-322300403941/crashlytics/app/android:org.odk.collect.android/issues/f2cab4a813a668ccdfa072f6c5d17d02?time=last-seven-days&types=crash&versions=v2022.3.1%20(4482)&sessionEventKey=62F2A20603C700014E1B2D7DCF0831FC_1708386372765513524) for example, right? In that case, the form index had not just changed so this block would not run?

Reply:
No, currently `handle` is being called so it must go through here.